tests/cc/test_zip.cc

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cstring>
+
+#include "bcc_zip.h"
+#include "catch.hpp"
+
+#define ENTRY_NAME "libdebuginfo_test_lib.so"
```

Comment:
Would it make sense to have a test for a hierarchical name as well, such as `/lib/arm64-v8a/libtest.so` that you mentioned in the pull request?

Reply:
Good idea. I added tests with a file in a subdirectory.